docs/contracts.rst

```diff
@@ -1035,8 +1034,7 @@ more advanced example to implement a set).
             // The library functions can be called without a
             // specific instance of the library, since the
             // "instance" will be the current contract.
-            if (!Set.insert(knownValues, value))
-                throw;
+            assert(Set.insert(knownValues, value));
```

Comment:
Wasn't the definition that `require()` is for input validation, while `assert()` is for everything else?